### ISSUE TITLE
Bug - Latest OpenUPM version is 1.0.2

### ISSUE THREAD

**Issue by Razenpok** [WooshiiDev/HierarchyDecorator#47]
The package manager shows me that there's an update even though the "1.0.2" version was released a heck while ago

![image](https://user-images.githubusercontent.com/8434406/149910537-3407700d-3b92-482b-ac6d-7c0cd957af29.png)

The problem is with the published versions, somehow 1.0.2 was published at some point

https://openupm.com/packages/com.wooshii.hierarchydecorator/?subPage=versions

You should probably contact the openupm maintainer to take down this version

**Comment by favoyang** [comment]
I guess the 1.0.2 has been published at a very early point. Then you removed that Git tag, and reset the version from 0.x.x. As a result, the 1.0.2 stays as the "latest", and all other "previous" versions have been tagged as "patch".

Anyway, version 1.0.2 has been unpublished. The latest tag is pointed to 0.8.4.

```
com.wooshii.hierarchydecorator@0.8.4 | proprietary or unlicensed | versions: 6

HierarchyDecorator
HierarchyDecorator is an extension for the Unity Hierarchy enhancing it's utility.
keywords: hierarchy, editor

dist
.tarball: https://package.openupm.com/com.wooshii.hierarchydecorator/-/com.wooshii.hierarchydecorator-0.8.4.tgz
.shasum: c340c39106b886cccf6541eeeedb60d0befda793
.integrity: sha512-vmUeYX8j9w46CXqOkdNJapo0aM7fC5CaZ/OJZjnjQ9MtmvGwNbqHBLzGB4Xr96eLgFcodQpGTivO34cHD4vtgQ==

latest: 0.8.4

published at 2022-01-17T07:07:04.881Z

versions:
  0.7.1
  0.8.0
  0.8.1
  0.8.2
  0.8.3
  0.8.4
```

**Comment by WooshiiDev** [comment]
Thanks for bringing this to my attention Razen, I always appreciate your issue posts 👏 

Thank you so much favoyang, and thank you for the quick response to this, my fault for being negligent in the past. Will make sure to keep on top in the future.
